components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { useMemo } from "react";
+import { useLocation } from "react-router-dom";
+import ContextMenu, { ContextMenuEntry } from "../components/ContextMenu";
+import { useCurrentTeam, useTeamMemberInfos, useTeams } from "../teams/teams-context";
+import { useCurrentUser } from "../user-context";
+
+export interface OrganizationSelectorProps {}
+
+export default function OrganizationSelector(p: OrganizationSelectorProps) {
+    const user = useCurrentUser();
+    const teams = useTeams();
+    const team = useCurrentTeam();
+    const teamMembers = useTeamMemberInfos();
+    const location = useLocation();
+
+    const userFullName = user?.fullName || user?.name || "...";
+    const entries: ContextMenuEntry[] = useMemo(
+        () => [
+            ...(!user?.additionalData?.isMigratedToTeamOnlyAttribution
+                ? [
+                      {
+                          title: userFullName,
+                          customContent: (
+                              <div className="w-full text-gray-500 flex flex-col">
+                                  <span className="text-gray-800 dark:text-gray-100 text-base font-semibold">
+                                      {userFullName}
+                                  </span>
+                                  <span className="">Personal Account</span>
+                              </div>
+                          ),
+                          active: team === undefined,
+                          separator: true,
+                          link: `${location.pathname}?org=0`,
+                      },
+                  ]
+                : []),
+            ...(teams || [])
+                .map((t) => ({
+                    title: t.name,
+                    customContent: (
+                        <div className="w-full text-gray-400 flex flex-col">
+                            <span className="text-gray-800 dark:text-gray-300 text-base font-semibold">{t.name}</span>
+                            <span className="">
+                                {!!teamMembers[t.id]
+                                    ? `${teamMembers[t.id].length} member${teamMembers[t.id].length === 1 ? "" : "s"}`
+                                    : "..."}
+                            </span>
+                        </div>
+                    ),
+                    active: team?.id === t.id,
+                    separator: true,
+                    link: `${location.pathname}?org=${t.id}`,
+                }))
+                .sort((a, b) => (a.title.toLowerCase() > b.title.toLowerCase() ? 1 : -1)),
+            {
+                title: "Create a new organization",
```

Comment:
**question:** Is there a value of showing the org selector when creating a new org? Could be fine to leave it as is for now.

| Navigation when creating an org |
|-|
| <img width="1440" alt="Screenshot 2023-02-02 at 5 52 54 PM (2)" src="https://user-images.githubusercontent.com/120486/216374005-1663fdab-d077-4efc-be38-79f5a1c86801.png"> |

Reply:
> Is there a value of showing the org selector when creating a new org?

I personally like consistency, and :+1: for keeping it.